python/lsst/pipe/tasks/hips.py

```diff
@@ -948,16 +950,7 @@ def run(self, bands, max_order, hips_exposure_handle_dict, do_color=False):
                 Q=self.config.png_gray_asinh_softening,
             )
         else:
-            png_color_mapping = AsinhMapping(
-                self.config.png_color_asinh_minimum,
-                self.config.png_color_asinh_stretch,
-                Q=self.config.png_color_asinh_softening,
-            )
-
-            bcb = self.config.blue_channel_band
-            gcb = self.config.green_channel_band
-            rcb = self.config.red_channel_band
-            colorstr = f"{bcb}{gcb}{rcb}"
+            colorstr = "".join(bands)
```

Comment:
It is possible for bands to be more than one color (e.g. the narrow-band filters for HSC).  Would this cause a problem in those cases?